core/src/kura.rs

```diff
@@ -335,6 +352,14 @@ impl Kura {
             .lock()
             .push((block.hash(), Some(Arc::new(block))));
     }
+
+    /// Replace the block in kura's in memory block store. Kura will write the block
+    /// to disk in due time.
```

Comment:
```suggestion
    /// Replace the block in `Kura`'s in-memory block store. 
```

*In due time* isn't particularly informative, and relies on external code. Being terse is often good for maintainability.